<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/pengui/icons/pengui.svg</normaloff>:/pengui/icons/pengui.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelLogo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>1024</width>
      <height>1024</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/pengui/logo/penguin-dad-2.jpeg</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuCalamares">
     <property name="title">
      <string>&amp;Calamares</string>
     </property>
     <addaction name="actionCalamaresInstall"/>
     <addaction name="actionCalamaresRemove"/>
    </widget>
    <addaction name="menuCalamares"/>
    <addaction name="action_Produce"/>
    <addaction name="action_Kill"/>
    <addaction name="actionStatus"/>
    <addaction name="actionCUckoo"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Configure"/>
    <addaction name="actionConfigureTools"/>
    <addaction name="actionFormatUSB"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionUsersGuide"/>
    <addaction name="actionBlog"/>
    <addaction name="separator"/>
    <addaction name="actionRepository"/>
    <addaction name="actionTelegram"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuPPA">
     <property name="title">
      <string>&amp;PPA</string>
     </property>
     <addaction name="actionIPpaAdd"/>
     <addaction name="actionPpaRemove"/>
    </widget>
    <addaction name="action_Clean"/>
    <addaction name="menuPPA"/>
    <addaction name="action_Skel"/>
    <addaction name="action_Yolk"/>
   </widget>
   <widget class="QMenu" name="menuWardrobe">
    <property name="title">
     <string>&amp;Wardrobe</string>
    </property>
    <addaction name="actionGet"/>
    <addaction name="actionList"/>
    <addaction name="actionShow"/>
    <addaction name="actionWear"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuWardrobe"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Produce">
   <property name="text">
    <string>&amp;Produce</string>
   </property>
  </action>
  <action name="action_Kill">
   <property name="text">
    <string>&amp;Kill</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Configure">
   <property name="text">
    <string>&amp;eggs,yaml</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Skel">
   <property name="text">
    <string>&amp;Skel</string>
   </property>
  </action>
  <action name="action_Clean">
   <property name="text">
    <string>&amp;Clean</string>
   </property>
  </action>
  <action name="action_Yolk">
   <property name="text">
    <string>&amp;Yolk</string>
   </property>
  </action>
  <action name="actionGet">
   <property name="text">
    <string>&amp;Get</string>
   </property>
  </action>
  <action name="actionList">
   <property name="text">
    <string>&amp;List</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>&amp;Show</string>
   </property>
  </action>
  <action name="actionWear">
   <property name="text">
    <string>&amp;Wear</string>
   </property>
  </action>
  <action name="actionRepository">
   <property name="text">
    <string>&amp;Repository</string>
   </property>
  </action>
  <action name="actionBlog">
   <property name="text">
    <string>&amp;Blog</string>
   </property>
  </action>
  <action name="actionUsersGuide">
   <property name="text">
    <string>&amp;Users' guide</string>
   </property>
  </action>
  <action name="actionTelegram">
   <property name="text">
    <string>&amp;Telegram</string>
   </property>
  </action>
  <action name="actionConfigureTools">
   <property name="text">
    <string>&amp;tools.yaml</string>
   </property>
  </action>
  <action name="actionCUckoo">
   <property name="text">
    <string>C&amp;uckoo</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>&amp;Status</string>
   </property>
  </action>
  <action name="actionCalamaresInstall">
   <property name="text">
    <string>Install</string>
   </property>
  </action>
  <action name="actionCalamaresRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionIPpaAdd">
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </action>
  <action name="actionPpaRemove">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
  </action>
  <action name="actionFormatUSB">
   <property name="text">
    <string>Format USB</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
